debezium-ddl-parser/src/main/java/io/debezium/ddl/parser/oracle/PlSqlLexerBase.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.ddl.parser.oracle;
+
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.Lexer;
+
+public abstract class PlSqlLexerBase extends Lexer {
+    public PlSqlLexerBase(CharStream input) {
+        super(input);
+    }
+
+    protected boolean IsNewlineAtPos(int pos) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
The code is called form `Lexer` is it might make more sense to confor to lexer rules than to Java rules.